components/script/dom/bindings/error.rs

```diff
@@ -304,7 +309,7 @@ pub(crate) unsafe fn report_pending_exception(
     }
 
     if dispatch_event {
-        GlobalScope::from_context(cx, realm).report_an_error(error_info, value.handle(), can_gc);
+        GlobalScope::from_context(*cx, realm).report_an_error(error_info, value.handle(), can_gc);
```

Comment:
Let's call GlobalScope::from_safe_conrext here instead.

Reply:
Thanks for the review, I have updated it to use `GlobalScope::from_safe_context`.